analyzer/compliance-analyzer/src/main/java/eu/fasten/analyzer/complianceanalyzer/ReplacedGCPAuthenticator.java

```diff
@@ -0,0 +1,67 @@
+package eu.fasten.analyzer.complianceanalyzer;
+
+import com.google.auth.oauth2.AccessToken;
+import com.google.auth.oauth2.GoogleCredentials;
+import io.kubernetes.client.util.KubeConfig;
+import io.kubernetes.client.util.authenticators.Authenticator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.Date;
+import java.util.Map;
```

Comment:
There are some unused imports

Reply:
Solved in https://github.com/fasten-project/fasten/commit/7975246450da52793713310dc6123c14c876daf5.